app/code/Magento/Multishipping/etc/frontend/di.xml

```diff
@@ -44,6 +44,7 @@
     </type>
     <type name="Magento\Checkout\Controller\Cart">
         <plugin name="multishipping_clear_addresses" type="Magento\Multishipping\Model\Cart\Controller\CartPlugin" sortOrder="50" />
+        <plugin name="multishipping_disabler" type="Magento\Multishipping\Plugin\DisableMultishippingMode" sortOrder="50" />
```

Comment:
I guess instead of adding this plugin to an Abstract class, that may lead us into undesired behavior. We probably should do it directly on the change qty action, like this:

```
    <type name="Magento\Checkout\Controller\Cart\UpdateItemQty">
        <plugin name="multishipping_disabler" type="Magento\Multishipping\Plugin\DisableMultishippingMode" sortOrder="50" />
    </type>
```

What do you think?

Reply:
Has been updated